agent/internal/agent.go

```diff
@@ -348,6 +348,8 @@ func (a *agent) connectToMaster(ctx *actor.Context) error {
 
 	a.socket, _ = ctx.ActorOf("websocket", api.WrapSocket(conn, proto.AgentMessage{}, true))
 
+	// TODO: Discuss: we don't use the resource pool field of this message - the resource pool was
+	//       already defined when we established the connection to the master.
 	started := proto.MasterMessage{AgentStarted: &proto.AgentStarted{
 		Version: a.Version, Devices: a.Devices, ResourcePool: a.ResourcePool, Label: a.Label}}
```

Comment:
@shiyuann Can we change the AgentStarted message so that it doesn't include the resource pool? By the time that message is sent, the master already knows which resource pool the agent belongs to and has created an agent actor in that resource pool.

Reply:
yes feel free to remove this field.